model/src/main/kotlin/com/android/gradle/replicator/model/internal/filedata/FileData.kt

```diff
@@ -167,7 +157,7 @@ class AndroidResourcePropertiesAdapter(private val resourceType: String? = null)
                 qualifiers = qualifiers!!,
```

Comment:
something to do in a subsequent CL. I am not a big fan of using nullable and then the !! operator for what is expected to be present. Instead you should use a Builder class that has a bunch of setXXX() like for instance setResourceType(resourceType: String), and setFileData(fileData: List<Long>) and only check that all mandatory values are present when build() is called.

Reply:
This is simple enough that I can add it to this CL. Let me know what you think of my implementation